pkg/api/serialization_test.go

```diff
@@ -68,13 +70,18 @@ func roundTrip(t *testing.T, codec runtime.Codec, item runtime.Object) {
 		return
 	}
 
+	dataString := string(data)
+	if !strings.HasPrefix(dataString, "{") {
```

Comment:
serialization_proto_test just injects a new dimension to fuzz test (JSON + protobuf)